.github/workflows/leancode_markup-publish.yml

```diff
@@ -0,0 +1,36 @@
+name: leancode_markup publish
+
+on:
+  push:
+    tags: ['leancode_markup-v*']
+
+jobs:
+  publish:
+    name: Publish to pub.dev
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+
+    defaults:
+      run:
+        working-directory: packages/leancode_markup
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: 3.2
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.16.x'
+          cache: true
+          
+      - name: Publish
+        run: dart pub publish -f
```

Comment:
I think we now publish using the mobile-tools script (see enhanced_gradients-publish.yml for reference)